packages/compiler-cli/test/ngtsc/incremental_spec.ts

```diff
@@ -70,6 +70,10 @@ runInEachFileSystem(() => {
     });
 
     it('should rebuild components that have changed', () => {
+      env.tsconfig({
+        ivyTemplateTypeCheck: true,
+        strictTemplates: true,
+      });
```

Comment:
`ivyTemplateTypeCheck` shouldn't be relevant here.

```suggestion
      env.tsconfig({strictTemplates: true});
```